client/src/pages/Settings/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,97 @@
+import { Button, Input, Typography } from '@mui/material';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { Box } from '@mui/system';
+import SettingHeader from '../../../components/SettingsHeader/SettingsHeader';
+import { User } from '../../../interface/User';
+import useStyles from './useStyles';
+import { useSnackBar } from '../../../context/useSnackbarContext';
+import { useState } from 'react';
+
+interface ProfilePhotoProps {
+  header: string;
+  currentUser?: User;
+  currentProfile?: any;
+}
+
+const ProfilePhoto: React.FC<ProfilePhotoProps> = ({ header, currentUser, currentProfile }) => {
+  const classes = useStyles();
+  const [isAttached, setIsAttached] = useState(false);
+  const { updateSnackBarMessage } = useSnackBar();
+
+  const attachFile = () => {
+    setIsAttached(true);
+    updateSnackBarMessage('File attached and ready for upload');
+  };
+
+  return (
+    <Box
+      sx={{
+        width: 600,
+        margin: '0 auto',
+      }}
+    >
+      <SettingHeader header={header} />
+      <Box textAlign="center">
+        {currentProfile.photo ? (
+          <img src={currentProfile.photo} className={classes.photo} alt="Uploaded user profile photo" />
```

Comment:
Rather than using an HTML image element, we can use the MUI Avatar component here :)

Ref: https://mui.com/api/avatar/